tests/system-tests/rdscore/internal/rdscorecommon/metallb-traffic-segregation.go

```diff
@@ -0,0 +1,731 @@
+			RDSCoreConfig.HypervisorUser,
+			RDSCoreConfig.HypervisorPass,
+			RDSCoreConfig.MetalLBFRRContainerNameOne,
+			ipRoute)
+		Expect(err).ToNot(HaveOccurred(),
+			fmt.Sprintf("first external FRR container %s route check for the %s failed: %v",
+				RDSCoreConfig.MetalLBFRRContainerNameOne, ipRoute, err))
+	}
+
+	By("Make sure that 2nd external FRR container has learnt route to the 2nd deployment")
+
+	ipRouteList = []string{
+		fmt.Sprintf("%s/32", RDSCoreConfig.MetalLBLoadBalancerTwoIPv4),
+		fmt.Sprintf("%s/128", RDSCoreConfig.MetalLBLoadBalancerTwoIPv6),
+	}
+
+	for _, ipRoute := range ipRouteList {
+		err := verifyIPRouteBGP(
+			RDSCoreConfig.HypervisorHost,
+			RDSCoreConfig.HypervisorUser,
+			RDSCoreConfig.HypervisorPass,
+			RDSCoreConfig.MetalLBFRRContainerNameTwo,
+			ipRoute)
+		Expect(err).ToNot(HaveOccurred(),
+			fmt.Sprintf("second external FRR container %s route check for the %s failed: %v",
+				RDSCoreConfig.MetalLBFRRContainerNameTwo, ipRoute, err))
+	}
+
+	By("Creating the tcpdump deployment for the first FRR packets capturing")
+
+	glog.V(rdscoreparams.RDSCoreLogLevel).Infof("Create tcpdump deployment for the node %v", lbOneNodeName)
+
+	_, err =
+		supporttools.CreateTCPDumpDeployment(
+			APIClient,
+			stNamespace,
+			stDeploymentLabel,
+			RDSCoreConfig.MetalLBSupportToolsImage,
+			RDSCoreConfig.MetalLBTrafficSegregationTCPDumpIntOne,
+			captureScript,
+			[]string{lbOneNodeName})
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to create supporttools deployment for the node %v in namespace %s from image %s: %v",
+			lbOneNodeName, stNamespace, RDSCoreConfig.MetalLBSupportToolsImage, err))
+
+	By("Finding support-tools pods")
+
+	glog.V(rdscoreparams.RDSCoreLogLevel).Infof("Searching for tcpdump pods in %q namespace", stNamespace)
+
+	stPodsList := findPodWithSelector(stNamespace, stDeploymentLabel)
+	Expect(len(stPodsList)).ToNot(Equal(0), "No tcpdump pods found in namespace %s",
+		stNamespace)
+
+	glog.V(rdscoreparams.RDSCoreLogLevel).Infof("Found %d 'traceRoute' pods", len(stPodsList))
+
+	By("Assert that 1st mockup app is reachable from the node with 1st FRR container")
+
+	timeStart := time.Now()
+
+	glog.V(rdscoreparams.RDSCoreLogLevel).Infof("start run traffic at %v", timeStart)
+
+	for _, appURL := range []string{bgpOneAppURLIPv4, bgpOneAppURLIPv6} {
+		err := verifyAppIsReachableFromFRRContainer(
+			RDSCoreConfig.HypervisorHost,
+			RDSCoreConfig.HypervisorUser,
+			RDSCoreConfig.HypervisorPass,
+			RDSCoreConfig.MetalLBFRRContainerNameOne,
+			appURL)
+		Expect(err).ToNot(HaveOccurred(),
+			fmt.Sprintf("first mockup app %s is not reachable from the container %s: %v",
+				appURL, RDSCoreConfig.MetalLBFRRContainerNameOne, err))
+	}
+
+	By("Check the traffic flows for the first FRR through the expected interface")
+
+	for _, _pod := range stPodsList {
+		searchString := "10.18.126.21"
+
+		glog.V(rdscoreparams.RDSCoreLogLevel).Infof("Making sure that the traffic flows for the first FRR "+
```

Comment:
shouldn't it be _from_
```suggestion
		glog.V(rdscoreparams.RDSCoreLogLevel).Infof("Making sure that the traffic flows from the first FRR "+
```